Pollo/Views/OptionsGradientView.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  OptionsGradientView.swift
+//  Pollo
+//
+//  Created by Gonzalo Gonzalez on 3/7/20.
+//  Copyright © 2020 CornellAppDev. All rights reserved.
+//
+
+import UIKit
+
+class OptionsGradientView: UIView {
+
+    // MARK: - View vars
+    let gradientLayer = CAGradientLayer()
+    
+    // MARK: - Constants
+    let animationDuration: TimeInterval = 0.2
+    let bottomInset: CGFloat = 9.8
+    let topOffset: CGFloat = 5
```

Comment:
make `private`

Reply:
some of the constants were actually not used; gradientLayer cannot be private because it's accessed on the outside to change the intensity of the gradient based on whether the user is admin/member.